src/main/java/com/hrznstudio/galacticraft/misc/capes/CapesLoader.java

```diff
@@ -24,53 +24,43 @@
 
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
-import com.hrznstudio.galacticraft.api.capes.CapeListener;
-import com.hrznstudio.galacticraft.api.capes.models.CapesModel;
+import com.hrznstudio.galacticraft.Galacticraft;
+import net.minecraft.util.Util;
 import org.apache.commons.io.IOUtils;
 
 import java.io.IOException;
 import java.net.URL;
 import java.nio.charset.Charset;
-import java.util.ArrayList;
-import java.util.List;
+import java.util.Map;
 
-/**
- * @author <a href="https://github.com/StellarHorizons">StellarHorizons</a>
- */
-public class CapeLoader {
-
-    private final List<CapeListener> listeners;
+public class CapesLoader {
+    private Map players;
```

Comment:
Is it possible to not use raw types (or is it required because of GSON)?

Reply:
yeah gson requires the raw type